src/components/NoteNameEditor/HeaderComponent/HeaderComponent.js

```diff
@@ -3,10 +3,13 @@ import PropTypes from 'prop-types';
 
 import { StyledButtonContainer, StyledHeader } from 'components/NoteNameEditor/HeaderComponent/HeaderStyles';
 
-export const HeaderComponent = ({ onDeleteClick, onUploadFileClick }) => (
+export const HeaderComponent = ({ onChangeParentClick, onDeleteClick, onUploadFileClick, changeNoteParentMode }) => (
```

Comment:
It's hard to understand if `changeNoteParentMode` is a function or a string or whatever.

If it's a boolean, let's start the variable name with `is`, e.g. `isChangeParentModeActive`.